src/stores/cluster/types.ts

```diff
@@ -174,20 +168,20 @@ export interface IBatchedClusterCreationErrorAction {
   error: string;
 }
 
-export interface IClusterDeleteConfirmedRequestAction {
-  type: typeof CLUSTER_DELETE_CONFIRMED;
-  cluster: Cluster;
+export interface IClusterDeleteRequestAction {
+  type: typeof CLUSTER_DELETE_REQUEST;
+  id: string;
 }
 
 export interface IClusterDeleteSuccessAction {
   type: typeof CLUSTER_DELETE_SUCCESS;
-  clusterId: string;
+  id: string;
   timestamp: string;
 }
 
 export interface IClusterDeleteErrorAction {
   type: typeof CLUSTER_DELETE_ERROR;
-  clusterId: string;
+  id: string;
```

Comment:
Switched up the interfaces a bit: this gives us automatic loading flags using the `entityloading` reducer.